_opensearch/install/rpm.md

```diff
@@ -0,0 +1,210 @@
+
+  ```
+  Fingerprint: c5b7 4989 65ef d1c2 924b a9d5 39d3 1987 9310 d3fc
+  ```
+
+  If correct, enter `yes`. The OpenSearch installation continues.
+
+7. Run OpenSearch and OpenSearch Dashboards using `systemctl`.
+
+  ```bash
+  systemctl start opensearch.service
+  systemctl start opensearch-dashboards.service
+  ```
+
+  If you configure the security plugin for production use (or disable it), you can run OpenSearch using `./bin/opensearch`.
+
+8. Open a second terminal session, and send requests to the server to verify that OpenSearch is running:
+
+  ```bash
+  curl -XGET https://localhost:9200 -u 'admin:admin' --insecure
+  curl -XGET https://localhost:9200/_cat/plugins?v -u 'admin:admin' --insecure
+  ```
+
+To stop running OpenSearch, enter:
+
+```bash
+systemctl stop opensearch.service
+systemctl stop opensearch-dashboards.service
+```
+
+
+## (Optional) Set up Performance Analyzer
+
+When enabled, the Performance Analyzer plugin collects data related to the performance of your OpenSearch instance. By default, the Performance Analyzer endpoints are not accessible outside the host machine.
+
+1. Make Performance Analyzer accessible outside of the host machine
+
+  ```bash
+  cd /usr/share/opensearch # navigate to the OpenSearch home directory
+  cd plugins/opensearch_performance_analyzer/pa_config/
+  vi performance-analyzer.properties
+  ```
+
+  Uncomment the line `#webservice-bind-host` and set it to `0.0.0.0`:
+
+  ```
+  # ======================== OpenSearch performance analyzer plugin config =========================
+
+  # NOTE: this is an example for Linux. Please modify the config accordingly if you are using it under other OS.
+
+  # WebService bind host; default to all interfaces
+  webservice-bind-host = 0.0.0.0
+
+  # Metrics data location
+  metrics-location = /dev/shm/performanceanalyzer/
+
+  # Metrics deletion interval (minutes) for metrics data.
+  # Interval should be between 1 to 60.
+  metrics-deletion-interval = 1
+
+  # If set to true, the system cleans up the files behind it. So at any point, we should expect only 2
+  # metrics-db-file-prefix-path files. If set to false, no files are cleaned up. This can be useful, if you are archiving
+  # the files and wouldn't like for them to be cleaned up.
+  cleanup-metrics-db-files = true
+
+  # WebService exposed by App's port
+  webservice-listener-port = 9600
+
+  # Metric DB File Prefix Path location
+  metrics-db-file-prefix-path = /tmp/metricsdb_
+
+  https-enabled = false
+
+  #Setup the correct path for certificates
+  certificate-file-path = specify_path
+
+  private-key-file-path = specify_path
+
+  # Plugin Stats Metadata file name, expected to be in the same location
+  plugin-stats-metadata = plugin-stats-metadata
+
+  # Agent Stats Metadata file name, expected to be in the same location
+  agent-stats-metadata = agent-stats-metadata
+  ```
+
+2. Restart the OpenSearch service:
+
+  ```bash
+  systemctl restart opensearch.service
+  ```
```

Comment:
There is no need to restart opensearch in order to use PA.
We should not need to modify PA either out of the box.

Reply:
Should I remove the the pa_config steps @peterzhuamazon?